gui/default/index.html

```diff
@@ -533,7 +533,7 @@ <h4 class="panel-title">
                       <tr>
                         <th><span class="fas fa-fw fa-share-alt"></span>&nbsp;<span translate>Shared With</span></th>
                         <td class="text-right">
-                          <span tooltip data-original-title="{{sharesFolder(folder)}}<br/>(&sup1; {{'The remote device has not accepted sharing this folder.' | translate}})" ng-bind-html="sharesFolder(folder)" />
+                          <span tooltip data-original-title="{{sharesFolder(folder)}} {{folderHasUnacceptedDevices(folder) ? '<br/>(<sup>1</sup>' + ('The remote device has not accepted sharing this folder.' | translate) + ')' : ''}}" ng-bind-html="sharesFolder(folder)" />
```

Comment:
I thought about moving it to a function, but the presentation context is not always the same.  E.g. we hide the whole paragraph with `ng-if` in the editing modal, but here in the tooltip it's a line break and parentheses to be hidden.

I don't think splitting into two spans can work.  Because we would need to split the *tooltip* content, which is only an attribute to the one span we have.

Reply:
Ah right, I wrongly assumed there ins't a tooltip at all without "not shared folders". Long line it is then :)